src/Powercord/plugins/pc-guilds/style.scss

```diff
@@ -73,6 +73,10 @@
   }
 }
 
+[data-react-beautiful-dnd-drag-handle="4"]{
```

Comment:
This attribute may change at any time

Reply:
Does the suggestions feature work?
I hope so
```suggestion
.pc-guild{
```

Reply:
Because one space good
```suggestion
.pc-guild {
```